language/tools/move-cli/src/sandbox/commands/run.rs

```diff
@@ -66,7 +66,7 @@ move run` must be applied to a module inside `storage/`",
 
     let signer_addresses = signers
         .iter()
-        .map(|s| AccountAddress::from_hex_literal(s))
+        .map(|s| AccountAddress::from_hex_fuzzy(s))
```

Comment:
maybe we should use the source language parsing here?

Reply:
And I was surprised to not a see a change for other arg parsing. Have you looked at that?

Reply:
I'm trying to scope down my stuff to AccountAddresses for now, we can come back and do other args later.